qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -469,18 +469,14 @@ def __repr__(self) -> str:
 
 
 class IBMQSimulator(IBMQBackend):
-    """Backend class interfacing with an IBMQ simulator."""
+    """Backend class interfacing with an IBM Quantum Experience simulator."""
 
     def properties(
             self,
             refresh: bool = False,
             datetime: Optional[python_datetime] = None
     ) -> None:
-        """Return the online backend properties.
-
-        Returns:
-            None
-        """
+        """Return the online backend properties."""
```

Comment:
I was mostly following the style for the `IBMQRetiredBackend` methods. Although the methods don't return anything, they still provide a description of the method. Would it be best to raise an exception as done in the [run](https://github.com/Qiskit/qiskit-ibmq-provider/blob/228bdf0be6b767880b8614a1848a0d543a7b3bf8/qiskit/providers/ibmq/ibmqbackend.py#L578) method?

Reply:
I was mostly following the style for the `IBMQRetiredBackend` methods. Although the methods don't return anything, they still provide a description of the method. Do you suggest removing the docstring?

Reply:
I didn't change the docstring because I thought it was there to provide a description, similar to the methods in `IBMQRetiredBackend`. Maybe something like the following would be better?
```
"""Return `None` since simulators do no have backend properties."""
```

Reply:
How about 
```
Return ``None`` since simulators do not have backend hardware properties. 
```